core/src/main/java/org/frankframework/receivers/PullingListenerContainer.java

```diff
@@ -392,27 +392,14 @@ public void run() {
 			}
 		}
 
-		private void rollBack(TransactionStatus txStatus, RawMessageWrapper<M> rawMessage, String reason) {
+		private void rollBack(TransactionStatus txStatus, String reason) {
 			if (log.isDebugEnabled()) {
 				String stackTrace = Arrays.stream(Thread.currentThread().getStackTrace())
 					.map(StackTraceElement::toString)
 					.reduce("\n", (acc, element) -> acc + "    at " + element + "\n");
 				log.debug("Rolling back TX, reason: {}, stack:{}", reason, stackTrace);
 			}
-			try {
-				txManager.rollback(txStatus);
```

Comment:
Weten we heel zeker dat de Receiver dit altijd doet (dus ook bij PullingListeners)?

Reply:
Ik denk dat ik een unit-test ga proberen te schrijven voor die PullingListenerContainer die verschillende flows kan simuleren, om te kijken of met alle mogelijke errors die optreden het record weer in status "Available" gezet gaat worden.

We hebben sowieso veel te weinig unit-test coverage hierop.